mmv1/products/compute/HaVpnGateway.yaml

```diff
@@ -41,6 +41,7 @@ async:
     resource_inside_response: false
 collection_url_key: 'items'
 custom_code:
+  post_create: 'templates/terraform/post_create/labels.tmpl'
```

Comment:
Is there a specific reason that we need this post_create?  From the [debug log](https://storage.cloud.google.com/ci-vcr-logs/beta/refs/heads/auto-pr-12866/artifacts/1020dae7-8c4e-4e43-a0d8-31ca8077a05f/recording/TestAccComputeHaVpnGateway_updateLabels.log), it looks like the initial POST call successfully created the labels.